app/src/main/java/eu/kanade/presentation/reader/PageIndicatorText.kt

```diff
@@ -35,6 +38,7 @@ fun PageIndicatorText(
     )
 
     Box(
+        modifier = Modifier.padding(bottom = 12.dp),
```

Comment:
This is not the way. You need to apply insets as padding instead of some arbitrary value. THo I've tried doing it but was unsuccessful.

Reply:
I'm also having issues with WindowInsets. I guess it's because we're hiding the bottom buttons of the phone, and this causes it not to behave as usual.

But I'll try a few more ways to work around this without using a fixed value for the padding.

Reply:
I couldn't come to a conclusion about which WindowInsets to use. We have `safeGestures` and `safeContent` that work, but I'm not sure if they are the most appropriate for this case. 

We also have `navigationBarsIgnoringVisibility`, which seems the most correct in this situation in my view, and it works very well when the phone is using the Navigation Bar with Swipe Gestures. However, when using the Navigation Bar with buttons, the padding becomes really large.

 Can you help me decide which direction to take?